### ISSUE TITLE
Rerun experiments with KnownModel.LDA ("asymmetric")

### ISSUE THREAD

**Issue by Alvant** [machine-intelligence-laboratory/OptimalNumberOfTopics#71]
- [x] 20NG
- [x] Brown
- [x] PN
- [x] Reuters
- [x] RuWikiGood
- [x] SO
- [x] WikiRef

**Comment by Alvant** [comment]
Prior fixed in #72

**Comment by Alvant** [comment]
```python
%%time

for model_family in [KnownModel.LDA]:
    template = PARAMS_EXPLORED[model_family]

    the_grid = [
            [[key, one_value] for one_value in template[key]]
            for key, params in template.items()
    ]

    print(model_family)

    for idx, model_params in enumerate(itertools.product(*the_grid)):

        # It is important to use the same index for "asymmetric" LDA
        if model_family != KnownModel.LDA or idx != 2 :
            continue
```
